### DISCUSSION TITLE
Layout handling

### DISCUSSION THREAD

**Discussion by SHA65536** [charmbracelet/bubbletea#307] [Q&A] [answered]
Hello!
Currently the hardest part of using BubbleTea is combining multiple bubbles on the screen at the same time.
While displaying bubbles one on top of another is fairly easy, stacking them horizontally proves tricky...
I found a community made [bubbleboxer]( https://github.com/treilik/bubbleboxer) that tries to solve this issue, but I constantly get crashes for the model being too wide / too high.
Do you guys have a default way to make these things work?
Cheers!

**Comment by bashbunni** [comment]
Hey SHA! 

What it sounds like you're looking for is `lipgloss.JoinHorizontal(lipgloss.<Position>, strs ...string)` or `lipgloss.JoinVertical(lipgloss.<Position>, strs ...string)`. I'm definitely going to add more layout examples, but in the meantime you can learn more about these methods [here](https://pkg.go.dev/github.com/charmbracelet/lipgloss#readme-joining-paragraphs)

**Comment by bashbunni** [comment]
Added details on layouts to the documentation on the `docs` branch

**Comment by treilik** [comment, marked as answer]
Hey SHA,
I am the creator of the `bubbleboxer`, sounds to me that your `Model`s don´t save the `bubbletea.WindowSizeMsg` and render there `View` accordingly.
But i would be happy about a issue if your problem still exists :)

**Reply by SHA65536** [reply]
Hey! Yeah the crashes were because of my misunderstanding of how the bubbles worked. Thanks for making that package btw.
Cheers:)
